contents/docs/session-replay/mobile.mdx

```diff
@@ -1,41 +1,72 @@
 ---
-title: Mobile Recording
+title: Mobile replays
 ---
 
 > 🚧 **NOTE:** Mobile recording is currently **only available on Android and iOS** and is considered `beta`. We are keen to gather as much feedback as possible so if you try this out please let us know. You can send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asession_replay%3Alow) or one of our other [support options](/docs/support-options).
 
-
-# Overview
-
-Mobile Session Recording allows you to record user sessions on mobile devices. This includes screen recordings, network requests, logs, and touches. This data can then be used to understand how users are interacting with your app and to identify bugs and issues.
+Mobile replays allows you to record user sessions in mobile apps. This includes screen recordings, network requests, logs, and touches. This data can then be used to understand how users are interacting with your app and to identify bugs and issues.
 
 ## How it works
 
 We have taken our time to make sure we provide a useful and detailed recording experience whilst keeping the performance and security of your app as a top priority. By default, the configuration is restrictive with automatic masking.
 
-#### Wireframe mode (Default)
+### Wireframe mode (default)
 
 Mobile recording is primarily done using native APIs to grab the view hierarchy state when the screen is drawn. This is done carefully so as not to affect performance in any way a user may notice.
 
 The view hierarchy is transformed to a JSON data structure and later rendered as an HTML [wireframe](https://www.figma.com/resource-library/what-is-wireframing/). Since it is a wireframe, the UI won't have the original look and feel but it should be close enough to understand the user's behavior.
 
-#### Screenshot mode
+Below is an example of a screen captured in wireframe mode in Android:
+
+![Wireframe mode in mobile replay](https://res.cloudinary.com/dmukukwp6/image/upload/v1722436140/posthog.com/contents/Screenshot_2024-07-31_at_3.27.44_PM.png)
 
-If the `screenshot` (Android) or `screenshotMode` (iOS) option is enabled, the SDK will take a screenshot of the screen instead of making a representation of the user's screen (A.k.a. Wireframe mode). The screenshot may contain sensitive information, use it with caution. The automatic masking for the screenshot mode already works on Android and we'll be working on the iOS next.
+### Screenshot mode
 
-#### Network recording
+If  `screenshot` (Android) or `screenshotMode` (iOS) option is enabled, the SDK will take a screenshot of the screen instead of making a wireframe representation of the user's screen. The screenshot may contain sensitive information, so proceed with caution and ensure you're masking all relevant views.
```

Comment:
```suggestion
If `screenshot` (Android) or `screenshotMode` (iOS) option is enabled, the SDK will take a screenshot of the screen instead of a wireframe representation. The screenshot may contain sensitive information, so proceed with caution and ensure you're masking all relevant views.
```